src/screens/GroupInfo/index.js

```diff
@@ -26,10 +26,10 @@ import ModalAction from './elements/ModalAction';
 import ModalActionAnonymous from './elements/ModalActionAnonymous';
 import ReportGroup from '../../assets/images/report.png';
 import useGroupInfo from './hooks/useGroupInfo';
+import {COLORS} from '../../utils/theme';
 import {Loading} from '../../components';
 import {ProfileContact} from '../../components/Items';
 import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed here. While this doesn't affect the functionality of the code, it's generally a good practice to maintain a consistent order of imports for better readability and maintainability. In this case, it seems like the `COLORS` import from `../../utils/theme` was moved up. If there's no specific reason for this change, I would suggest reverting it back to its original position.
```diff
- import {COLORS} from '../../utils/theme';
- import {Loading} from '../../components';
- import {ProfileContact} from '../../components/Items';
- import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
+ import {Loading} from '../../components';
+ import {ProfileContact} from '../../components/Items';
+ import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->